xcode/Subconscious/Shared/Services/AddressBookService.swift

```diff
@@ -39,25 +39,17 @@ extension AddressBookError: LocalizedError {
     }
 }
 
-actor AddressBookService {
-    private var noosphere: NoosphereService
-    private var database: DatabaseService
+/// An AddressBook can wrap any Sphere and provide a higher-level interface to manage petnames.
+actor AddressBook<Sphere: SphereProtocol> {
```

Comment:
I broke apart the `AddressBook` and `AddressBookService` so I could re-use `listEntries` in the `UserProfileService` without having to pass a `database` dependency around.